python/py-terminaltables/Portfile

```diff
@@ -5,7 +5,7 @@ PortGroup           python 1.0
 
 name                py-terminaltables
 version             3.1.10
-revision            0
+revision            1
```

Comment:
it seems like this is only fixing a build issue for the PY311 port that you added here (as that's the first one that defaults to `python.pep517 yes`). So there is no need to increase the `revision` in that case as ports that did build before will not be affected by this change.

Reply:
Heya, the build also fails on previous versions (I'm intentionally trying the builds on py310, not py311). hence the fix.

Reply:
okay, but it is still a build failure and this a "revision" increase is not needed.